manifests/04-openshift-logging.Namespace.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-logging
```

Comment:
Do we need to create this namespace here? Shouldn't it be created by the cluster-logging-operator?